src/components/Alumni/Alumni.jsx

```diff
@@ -0,0 +1,30 @@
+import React, {useState} from "react";
+import AlumniCarousel from "./AlumniCarousel";
+
+function Alumni() {
+    return (
+        <section className="my-[6.87rem] flex flex-col items-center gap-y-6">
+            <AlumniHeader/>
+            <AlumniCarousel/>
+            <SeeAllFeature className="relative bottom-12"/>
+        </section>
+    )
+}
+
+function AlumniHeader() {
+    return (
+        <div  className="flex flex-col items-center gap-y-4">
+            <h1 className="text-h1 font-header">Careers - where are our alumni?</h1>
+            <h3 className="text-h3">Our programs prepare you for a career as product manager! </h3>
+        </div>
+    )
+}
+
+function SeeAllFeature({className}) {
```

Comment:
I can take this out and make this into a shared component in my next MR

Reply:
Sounds good. Should I leave it as is rn?

Reply:
Yep let's leave it